data/rat.lean

```diff
@@ -954,6 +954,10 @@ lemma num_ne_zero_of_ne_zero {q : ℚ} (h : q ≠ 0) : q.num ≠ 0 :=
 assume : q.num = 0,
 h $ zero_of_num_zero this
 
+@[simp] lemma num_one : (1 : ℚ).num = 1 := rfl
```

Comment:
Should these two simp-lemmas be generalised to arbitrary integers?

Reply:
They already have been. `coe_int_num`. However `coe_int_one` won't simplify this expression, so this is necessary as well.